### ISSUE TITLE
Some calculations not rounding properly.

### ISSUE THREAD

**Issue by KiaSkretteberg** [ThomasHambach/NcalcJS#11]
I noticed that the library seems to occasionally be rounding incorrectly. I initially thought this was due to using floating point math but after re-looking at the values I'm not positive. This is my initial question/observation, but feel free to just skip ahead to the example below:

> I am using this library for doing calculations for an online point of sale and it's extremely important that the math works out correctly. I have tried performing the same calculation using the C# version of the NCalc library and it performs the calculation correctly. It's extremely inconvenient for me to do the calculations in C# though as I would much rather do them in the browser as the user types as it's much faster. 
>
> Is there any way to resolve this discrepancy in the library? I believe if all calculations were done as integer math and converted to decimals when evaluated, that would solve the issue, though I'm not positive. I was attempting to manually do this myself by modifying my expressions to multiply values by 10 to make them integers and then divide by 10 as the last step in the expression but it's proving to be extremely complicated. If this could be handled internally by the library that would be super beneficial.

Here is an example that I am running into issues with:

If my expression is `[b] * 0.1` and `b = 147.35`, the expression evaluates to `14.73`. It should evaluate to `14.74`. Is there a reason for this incorrect rounding? Is there a setting for it? Is there something I can do to fix this or is this a bug in the library?

Edit: As a side note, I looked at the library quickly to see if there was maybe an answer to this already and I saw that "Round does not not return the correct value" -- is this related?

**Comment by ThomasHambach** [comment]
It definitely is related and have not gotten around to fix it. If you're able to make a PR that would be awesome. I don't think I have the time in the next weeks to look at it. If not, I'll follow up later. Thanks for bringing it up. 

Even a unit test with your case failing would help me figure out the root cause.

**Comment by KiaSkretteberg** [comment]
While trying to figure out where this was going wrong in the ncalcjs library I realized that my issue was actually outside of the library (I'm not manually calling `Round` anywhere) and had to do with calling toFixed after performing the calculation - toFixed was not rounding 0.5 the way I was expecting. This is a common issue online however and I ended up using the function given here https://stackoverflow.com/a/23560569 which performs the rounding as I expect before calling the toFixed function. I did however realize there appears to be a setting in the ncalcjs library for rounding away from zero (likely copied from the .Net NCalc implementation) that is not being used. Perhaps this could be used in conjunction with a solution like the one I linked in order to properly implement the Round function (which would suffer from the same toFixed hurdles I encountered).
